assets/helpers/abtest.js

```diff
@@ -143,8 +155,8 @@ function getParticipation(mvtId: number): Participations {
 
 export const init = () => {
 
-  const mvt = getMvtId();
-  let participation = getParticipation(mvt);
+  const mvt: number = getMvtId();
```

Comment:
Why do you need to make the type explicit here?  Does flow not do type inference?